p3/dataaccess.py

```diff
@@ -265,3 +270,27 @@ def tags():
 tags = cache_me(
     signals=(cdata.tags.invalidated,),
     models=(models.P3Profile, cmodels.AttendeeProfile))(tags)
+
+def tags_for_proposed_talks():
```

Comment:
Ottimo, però io chiamerei la funzione tags_for_conference_talks e non mi limiterei ai talk "proposed".

Una funzione che restituisce tutti i tag usati in una conferenza la vedo più utile di una che lavora solo su i talk proposti (in questo caso potremmo usarla solo durante il voting).

Aggiungerei due parametri, il codice della conferenza (obbligatorio, ci penserà il template tag a fornire come default la conferenza attuale) e lo stato dei talk (default None, che significa tutti)